language-design.md

```diff
@@ -51,6 +95,10 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+Names are important, specifics matter. Before breifly learning about API design in the Bloch reading, I would have thought that this
+discussion was silly and not very important. However, I can now see why people may take different stances on this topic and determine
+a standard for people to agree on. API design must keep the general public in mind. Even if seems the best to you, make sure others agree.
```

Comment:
Some programs have some whacky syntax forever cemented in their language and I can only imagine these came about from not enough consulting of others. If you don't keep the public in mind, the language can be too much trouble to use.